src/frontend/scss/components/_subheader.scss

```diff
@@ -31,6 +31,19 @@ $r-subheader-search-title-width: 19rem !default; // aligned on computed search r
     }
   }
 
+  // Main document title
+  &__code {
+    @include make-container();
+    @include make-container-max-widths();
+    @include sv-flex(1, 0, 100%);
+
```

Comment:
Just a nitpick. As code is not an important information for user, I think it should be visually discreet.

```css
margin-top: -0.5rem;
margin-bottom: 0.5rem;
```

By using this vertical margin, Reference code will be visually linked to the course title. So visually these two elements appears like a one block.


![discreet-ref](https://user-images.githubusercontent.com/9265241/105039235-98b08e80-5a60-11eb-909c-f82ae274e6c2.jpg)

Reply:
Nice suggestion. I applied it.